Symfony/CS/Fixer/Contrib/CombineConsecutiveUnsetsFixer.php

```diff
@@ -83,8 +83,8 @@ public function getDescription()
      */
     public function getPriority()
     {
-        // should ran before SpacesAfterSemicolonFixer, WhitespacyLinesFixer, TrailingSpacesFixer and ExtraEmptyLinesFixer.
-        return 25;
+        // should ran before SpacesAfterSemicolonFixer, WhitespacyLinesFixer, TrailingSpacesFixer and ExtraEmptyLinesFixer and after NoEmptyStatementFixer.
+        return 24;
```

Comment:
It's always hard when writing fixers to work out what number to put there without checking a load of other files first.